extractor.go

```diff
@@ -460,6 +548,44 @@ func mapValidator(v abci.ValidatorUpdate) (*codec.Validator, error) {
 	}, nil
 }
 
+func mapValidators(v []*types.Validator) ([]*codec.Validator, error) {
+	validators := []*codec.Validator{}
+	if len(v) > 0 {
```

Comment:
Dont need the `len` check in here. Also, since we're not doing any conditional checks in the `for` loop, you can allocated the validators slice with the exact capacity (`make(...)`)